docs/organizations/verify-user-permissions.mdx

```diff
@@ -48,9 +48,11 @@ The examples below work for both SSR and CSR. Examples are written for Next.js A
   </Tab>
 
   <Tab>
-    The `useAuth()` hook can help you access what permissions a user has by using `has()`. In the example below, `has({ permission: "org:team_settings:manage" })` returns `false` if the user does not have the `org:team_settings:manage` permission.
+    Use the [`useAuth()`](/docs/references/react/use-auth) hook to access the `has()` helper in Client Components.
 
-    ```tsx filename="/app/dashboard/settings/form.tsx"
+    The following example uses `has()` to inspect a user's permissions granularly. If the user doesn't have the `org:team_settings:manage` permission, `has()` returns `false`, causing the component to return `null` instead of rendering its children.
+
+    ```tsx {{ filename: '/app/dashboard/settings/form.tsx' }}
```

Comment:
There's nothing wrong with doing this, but also it's not necessary because we transform the old convention into the new ones behind the scenes ([discussion](https://clerkinc.slack.com/archives/C06CA0JUZ8D/p1717697264843089?thread_ts=1717697144.373819&cid=C06CA0JUZ8D))

Reply:
yes but its removing tech debt as we move along